src/Explorer/Controls/Settings/SettingsUtils.tsx

```diff
@@ -131,13 +131,12 @@ export const parseConflictResolutionProcedure = (procedureFromBackEnd: string):
 };
 
 export const getSanitizedInputValue = (newValueString: string, max: number): number => {
```

Comment:
So the scenario this is for is like this:
Max is 100
user input 9 in the textfiled -> maintained as 9
user inputs another 9 in the textfield -> maintained as 99
user inputs another 9 in the textfield -> exceeds max of 100, so keep it as 99.

I think we should retain the behavior of chopping off the most recent input instead of maxing it min(max, input). It will be jumpy otherwise

Reply:
Discussed with Srinath offline. We are going to stick with this behavior for now.